### PR TITLE
Added the Gallazzi+05 stellar ages data

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
Does what it says.

### CODE REVIEW

File: data/GalaxyStellarMassStellarAges/conversion/convertGallazzi2005.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    exec(handle.read())
+
+# Cosmology
+h_sim = cosmology.h
+h_obs = 0.7
+
+input_filename = "../raw/Gallazzi_2005_ascii.txt"
+delimiter = "\t"
+
+output_filename = "Gallazzi2005_Age_Data.hdf5"
```
Comment (JBorrow) [inline data/GalaxyStellarMassStellarAges/conversion/convertGallazzi2005.py:19]: Remove _Age_ here - this doesn't fit with the naming scheme for other files

Comment (MatthieuSchaller) [reply data/GalaxyStellarMassStellarAges/conversion/convertGallazzi2005.py:19]: Ok. I added it since you already have a Gallazzi2005_Data for the Z.

Comment (JBorrow) [reply data/GalaxyStellarMassStellarAges/conversion/convertGallazzi2005.py:19]: I think it's best to let these be delimited by the directories

Comment (MatthieuSchaller) [reply data/GalaxyStellarMassStellarAges/conversion/convertGallazzi2005.py]: Done.

**Comment by JBorrow** [comment]
Thanks!

**Pull request merged**